content/tutorials/manage-the-rippled-server/installation/build-run-rippled-macos.md

```diff
@@ -69,6 +69,10 @@ For development purposes, run `rippled` as a non-admin user, not using `sudo`.
 
     Or, you can checkout one of the tagged releases listed on [GitHub](https://github.com/ripple/rippled/releases).
 
+0. Check the commit log to be sure you're compiling the right code. The most recent commit should be signed by a well-known Ripple developer and should set the version number to the latest released version. The [release announcements for `rippled`](https://xrpl.org/blog/label/rippled-release-notes.html) generally show the exact commit to expect for that release.
```

Comment:
> a well-known Ripple developer

How would a user not familiar with the community know who these developers are? Perhaps we should link to the rippled repo contributors or something similar to help people make the connection.

Reply:
Yeah, I don't have a bulletproof solution for how people know who signs the commits. In the past year or so 5 different people (including one who's no longer at Ripple) have signed release commits. The release announcement at least shows the name and commit ID (which is a cryptographic hash) so that much is reliable.